universe/resource.json

```diff
@@ -1,7 +1,7 @@
 {
   "assets": {
     "uris": {
-      "dcos-ui-bundle": "{{artifact-dir}}/release.tar.gz"
+      "dcos-ui-bundle": "{{artifact-dir}}/release-{{package-version}}.tar.gz"
```

Comment:
We need to follow a similar naming schema we use for our bumps. Orherwise we’ll run into issues with different releases because version numbers arent unique. 🤷‍♂️

Reply:
I’d like to see the same filename actually. Keep stuff consistent

Reply:
👍